From 6bdcf53de74ac2afba42deea63522939ca51f871 Mon Sep 17 00:00:00 2001
From: Raphael Robatsch <raphael-git@tapesoftware.net>
Date: Mon, 25 Dec 2023 16:15:29 +0000
Subject: [PATCH] Do not forcibly set OPENSSL_ROOT_DIR.

CMake can already find OpenSSL via pkg-config. Setting OPENSSL_ROOT_DIR
forcibly to "/usr" breaks this.
---
 CMakeLists.txt | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 288bcbe8..9750fae6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -119,17 +119,6 @@ include(.CMake/compiler_opts.cmake)
 include(.CMake/alg_support.cmake)
 
 if(${OQS_USE_OPENSSL})
-    if(NOT DEFINED OPENSSL_ROOT_DIR)
-        if(${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Darwin")
-            if(EXISTS "/usr/local/opt/openssl@1.1")
-                set(OPENSSL_ROOT_DIR "/usr/local/opt/openssl@1.1")
-            elseif(EXISTS "/opt/homebrew/opt/openssl@1.1")
-                set(OPENSSL_ROOT_DIR "/opt/homebrew/opt/openssl@1.1")
-            endif()
-        elseif(${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Linux")
-            set(OPENSSL_ROOT_DIR "/usr")
-        endif()
-    endif()
     find_package(OpenSSL 1.1.1 REQUIRED)
 endif()
 
-- 
2.42.0

